source/simulator/parameters.cc

```diff
@@ -1046,7 +1046,7 @@ namespace aspect
                          "the ``prescribed field'' method. Afterwards, the field is diffused based on "
                          "a solver parameter, the diffusion length scale, smoothing the field. "
                          "Specifically, the field is updated by solving the equation "
-                         "$(I-\\nabla \\cdot l^2 \\nabla) C_\\text{smoothed} = C_\\text{prescribed}$, "
+                         "$(I-l^2 \\Delta) C_\\text{smoothed} = C_\\text{prescribed}$, "
```

Comment:
I like your other changes, but for this one, I just find `\\nabla \\cdot l^2 \\nabla` easier to read (as the temperature diffusion equation is written in the same way). Here, it doesn't make a difference because `l^2` is constant, but for our normal diffusion equation it does. 
But if you feel strongly about this, we can change it like you suggested.